components/button/index.css

```diff
@@ -294,6 +294,17 @@ a.spectrum-Button {
 	}
 }
 
+/* Disable button label wrap  */
+.spectrum-Button--noWrap {
+	justify-content: flex-start;
```

Comment:
With the `overflow` and `text-overflow` additions, is this addition still needed? With short text plus the minimum width and for buttons with some other set width, the text will no longer be centered:
![Screenshot 2024-09-16 at 3 10 40 PM](https://github.com/user-attachments/assets/e7975a26-6d78-4005-bbc4-c98d9c826a86)

Reply:
@jawinn When the max-inline-size is set all the button content gets cut off from left and right edges of the container. I was thinking of doing a flex-start and overflow additions it'll show the dev the text is too long and they need to make an adjustment character wise. 

After some thought, I'm going to remove this because it's not a design specification and I don't want teams to get mislead in anyway.

Reply:
I do think what you had, `.spectrum-Button-label { overflow: hidden; text-overflow: ellipsis; }` is a good idea. I think we _should_ handle the text overflow for cases where the button's width is constrained, as it gets a bit broken as seen in the Chromatic testing view right now. Does it work okay with just those two properties?

Reply:
@jawinn Yes, they work with just these two properties.